src/main/java/com/spotify/docker/client/messages/ContainerConfig.java

```diff
@@ -392,6 +400,9 @@ public static Healthcheck create(
       if (retries != null) {
         builder.retries(retries);
       }
+      if (startPeriod != null) {
```

Comment:
Yeah, that sounds about right. To look at it charitably, if I can identify other people's mistakes that I have also made myself, it means I am growing.